Finds more problems now:
```
Run ./scripts/coccinelle.sh
diff -u -p a/src/ts_catalog/chunk_column_stats.c b/src/ts_catalog/chunk_column_stats.c
--- a/src/ts_catalog/chunk_column_stats.c
+++ b/src/ts_catalog/chunk_column_stats.c
@@ -854,8 +854,10 @@ chunk_get_minmax(const Chunk *chunk, Oid
 				(errcode(ERRCODE_INTERNAL_ERROR),
 				 (errmsg("could not get the min/max values for column \"%s\" of chunk \"%s.%s\"",
 						 col_name,
-						 chunk->fd.schema_name.data,
-						 chunk->fd.table_name.data))));
+						 /* Use NameStr rather than accessing data member directly */
+						 NameStr(chunk->fd.schema_name),
+						 /* Use NameStr rather than accessing data member directly */
+						 NameStr(chunk->fd.table_name)))));
 	}
 
 	pfree(command.data);
Error: Process completed with exit code 1.
```

--- a/src/ts_catalog/chunk_column_stats.c
+++ b/src/ts_catalog/chunk_column_stats.c
@@ -854,8 +854,8 @@ chunk_get_minmax(const Chunk *chunk, Oid col_type, const char *col_name, Datum *
 				(errcode(ERRCODE_INTERNAL_ERROR),
 				 (errmsg("could not get the min/max values for column \"%s\" of chunk \"%s.%s\"",
 						 col_name,
-						 chunk->fd.schema_name.data,
-						 chunk->fd.table_name.data))));
+						 NameStr(chunk->fd.schema_name.data),
+						 NameStr(chunk->fd.table_name.data)))));
 	}
 
 	pfree(command.data);

--- a/src/ts_catalog/chunk_column_stats.c
+++ b/src/ts_catalog/chunk_column_stats.c
@@ -854,8 +854,8 @@ chunk_get_minmax(const Chunk *chunk, Oid col_type, const char *col_name, Datum *
 				(errcode(ERRCODE_INTERNAL_ERROR),
 				 (errmsg("could not get the min/max values for column \"%s\" of chunk \"%s.%s\"",
 						 col_name,
-						 NameStr(chunk->fd.schema_name.data),
-						 NameStr(chunk->fd.table_name.data)))));
+						 NameStr(chunk->fd.schema_name),
+						 NameStr(chunk->fd.table_name)))));
 	}
 
 	pfree(command.data);

